chord_diagram.py

```diff
@@ -210,11 +210,19 @@ def chord_diagram(mat, names=None, order=None, width=0.1, pad=2., gap=0.03,
     else:
         try:
             chord_colors = [ColorConverter.to_rgb(chord_colors)] * num_nodes
-        except ValueError:
-            assert len(chord_colors) == num_nodes, \
+        except ValueError: 
+            assert (len(chord_colors) % num_nodes) == 0, \
```

Comment:
this would let people set incorrect numbers of colors (e.g. 2 by node for a 3x3 matrix)

Reply:
Yes, that's what I meant: 6 passes the test but is not a valid number of colors for a 3x3 matrix